src/beazleybond/BodicePattern.java

```diff
@@ -244,9 +249,7 @@ protected void createBlocks()
                                         refTopLeft.getY() - Arb_HalfFrontNeckWidth - frontShoulderLineY);
         Vector2D preStartPt = new Vector2D(0.0, refTopLeft.getY() - Arb_HalfFrontNeckWidth);
 
-        // Compute end point for first curve
-        Vector2D endPt = new Vector2D((2.0 / 3.0) * (f_ArmholeDepth - frontShoulderLineX),
-                                      touchFront.getY());
```

Comment:
What has changed here?

Reply:
Slight changing of the endPt to get a more accurate output

Reply:
What does that mean? The touch point should be the exact point on the construction line the first curve ends. How does changing this value make anything more accurate?